src/applications/ivc-champva/10-7959a/config/form.js

```diff
@@ -237,7 +237,8 @@ const formConfig = {
           depends: formData =>
             get('hasOhi', formData) &&
             get('claimType', formData) === 'medical' &&
-            get('policies', formData).length > 1,
+            get('policies', formData) &&
+            formData?.policies?.length > 1,
```

Comment:
Checks if the array exists, and then uses conditional chaining to check if the length is over one. Previously, if the user hadn't actually added any policies we could get a console warning. This prevents that and makes the whole check more explicit.